README.md

```diff
@@ -29,15 +27,37 @@ Download the data files for a study folder, for example brca_tcga:
 git lfs pull -I public/brca_tcga
 ```
 
-## Download a complete MySQL export of the latest database
+## How to Upload Data
+#### Step1 -Create a new branch from the 'master' branch.
+```
+git checkout master
+git pull origin master
+git checkout -b [name_of_your_new_branch]
+```
+For general background on creating and managing branches within GitHub, see:  [Git Branching and Merging](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging).
+
+#### Step2 -Commit changes, and push the branch back to GitHub.
+```
+[back to the root directory]
+git add .
+git commit -m '[notes_for_your_change]'
+git push origin [name_of_your_new_branch]
+```
 
+#### Step3 -Open a Pull Request on GitHub to the 'master' branch.
+For instructions on submitting a pull-request, please see:  [Using Pull Requests ](https://help.github.com/articles/using-pull-requests/) and [Sending Pull Requests](http://help.github.com/send-pull-requests/).
+
+## Automated tests on Circle CI
+All Pull Requests are automatically tested on [Circle CI](https://circleci.com/).
+
+Validation status of latest studies added to this repo: [![CircleCI](https://circleci.com/gh/cBioPortal/datahub.svg?style=svg)](https://circleci.com/gh/cBioPortal/datahub)
+
```

Comment:
Nice documentation! Some minor comments:

> All Pull Requests are automatically tested on [Circle CI](https://circleci.com/).

You can add that we use the the validation code from the cBioPortal master branch.

> Validation status of latest studies added to this repo:....

We just changed this! See the updated README in master :)